packages/quantic/force-app/main/default/lwc/quanticFeedbackModal/quanticFeedbackModal.js

```diff
@@ -0,0 +1,215 @@
+// @ts-nocheck
+import cancel from '@salesforce/label/c.quantic_Cancel';
+import done from '@salesforce/label/c.quantic_Done';
+import explainWhy from '@salesforce/label/c.quantic_ExplainWhy';
+import fillOutField from '@salesforce/label/c.quantic_FillOutField';
+import selectOneOfOptions from '@salesforce/label/c.quantic_SelectOneOfOptions';
+import selectTheReason from '@salesforce/label/c.quantic_SelectTheReason';
+import sendFeedback from '@salesforce/label/c.quantic_SendFeedback';
+import thankYouForYourFeedback from '@salesforce/label/c.quantic_ThankYouForYourFeedback';
+import whatWasTheReason from '@salesforce/label/c.quantic_WhatWasTheReason';
+import yourResponsesHelp from '@salesforce/label/c.quantic_YourResponsesHelp';
+import {getBueno} from 'c/quanticHeadlessLoader';
+import LightningModal from 'lightning/modal';
+import {api} from 'lwc';
+import errorTemplate from './error.html';
+import feedbackFormTemplate from './feedbackForm.html';
+import successTemplate from './success.html';
+
+/**
+ * The `QuanticFeedbackModal` component overlays a message modal on top of the current app window, the modal contains a form that allows the user to give feedback.
+ *
+ * @category Search
+ * @category Insight Panel
+ */
+export default class QuanticFeedbackModal extends LightningModal {
+  /**
+   * @api
+   * The list of options to be displayed in the modal.
+   * @type {Array<{label: string, value: string}>}
+   */
+  @api options;
```

Comment:
I'm thinking this component might be a good candidate to configure options using a slot. It could be a nice way to support different option types, like regular options and comment options.

```html
<c-quantic-feedback-modal>
  <c-quantic-feedback-option value="did-not-answer" label="This didn't answer my question" />
  <c-quantic-feedback-comment-option value="partially-answer" label="This only partially answered my question" optional-comment />
  <c-quantic-feedback-comment-option value="other" label="Other" />
</c-quantic-feedback-modal>
```

Maybe it is too much. Let me know what you think of the idea.

Reply:
The only issue I found with this is that this component is not supposed to be include in the HTML template like all the the other LWC components.
This component is a bit special since it extends `LightningModal` instead of `LightningElement` and the we can use it only by importing it in a JS file and opening it like so: 
```   
await FeedbackModal.open({
      size: 'small',
      label: 'Example feedback modal',
      description: 'Example feedback modal',
      options: this.prepareOptions(),
      submitFeedback: () => {},
    });
```
We can pass props with this static method `open` but not sure we can pass slots.

Reply:
Good question Mehdi :) 
I would've also advocated for slots but this is a good point if you can't figure out where the slots go in this describe object, it's definitely okay to do it like you did.

Reply:
@mmitiche You are right, `LightningModal` does not seem to support slots. I didn't realize that, sorry.